packages/webapp/src/components/lists/ReportList/ReportOptions.tsx

```diff
@@ -240,9 +240,19 @@ export const ReportOptions: FC<ReportOptionsProps> = memo(function ReportOptions
 	}, [showFieldFilters, hiddenFields, setShowFieldFiltersSelected])
 
 	const defaultReportType = reportOptions.find((r) => r.value === type) ?? reportOptions[0]
+
+	const firstService = filterOptions
+		? filterOptions?.options
+			? filterOptions?.options[0]
```

Comment:
You can use `options?.[0]` to continue your optional chaining without a double if/else shorthand.

Reply:
not sure why I wasn't aware you could use `?.` with `[0]`, thank you for the tip!